installer/scripts/service_control

```diff
@@ -346,6 +367,15 @@ restart_all_omsagent()
        setup_variables ${ws_id}
        restart_omsagent
     done
+
+    for ws_id in $NON_OMS_WS
+    do
+        ls -1 $ETC_DIR | grep -w scom > /dev/null 2> /dev/null
```

Comment:
Ditto * 2 again, see above. There's a lot of code duplication here, should this be a common subroutine?